### PR TITLE
Hatem fastapi task

### PR DESCRIPTION
**Pull request by Hatem29**
Created a small api for task creation and helping, with features including :
* UUID for each task uniqely identifing each task, with a specified set of statuses
* Urgent task implementation that has an extra field of specified set of priorities
* Multiple endpoints for reading the tasks created including one for reading urgent tasks
* Endpoints for creating, deleteing, and updating tasks
* Error handling features specifing to the user when and why an error happens
* Special response for each endpoint leveraging http stataus codes

### CODE REVIEW

File: Hatem/requirements.txt
Comment (Mohammed-samaheen) [inline Hatem/requirements.txt:1]: Please ensure that you include the required packages in this file.

**Review by Mohammed-samaheen — CHANGES_REQUESTED** [review]

File: Hatem/requirements.txt
Comment (Mohammed-samaheen) [reply Hatem/requirements.txt:1]: You didn't resolve this part.

Comment (Hatem29) [reply Hatem/requirements.txt:1]: got no idea why it doesn't show, it is there locally and I checked the git ignore files, nothing specified

Comment (Mohammed-samaheen) [reply Hatem/requirements.txt:1]: gitignore will ignore the file itself. Could you please send me the details in the slack?

**Review by Mohammed-samaheen — APPROVED** [review]

**Pull request merged**